deployment/submit_emr_job_train.py

```diff
@@ -23,57 +23,56 @@
 from time import gmtime, strftime
 import daiquiri
 
-import recommendation_engine.config.cloud_constants as config
+# import recommendation_engine.config.cloud_constants as config
 
 daiquiri.setup(level=logging.DEBUG)
 _logger = daiquiri.getLogger(__name__)
 
 COMPONENT_PREFIX = "chester"
 
 
-def submit_job(input_bootstrap_file, input_src_code_file):
+def submit_job():
     """Submit new job with specified parameters."""
     str_cur_time = strftime("%Y_%m_%d_%H_%M_%S", gmtime())
-
     # S3 bucket/key, where the input spark job ( src code ) will be uploaded
-    s3_bucket = config.DEPLOYMENT_PREFIX + '-automated-analytics-spark-jobs'
-    s3_key = '{}_{}_tf_training_job.zip'.format(config.DEPLOYMENT_PREFIX, COMPONENT_PREFIX)
+    s3_bucket = 'cvae-insights'
+    s3_key = 'tf_training_job.zip'
     s3_uri = 's3://{bucket}/{key}'.format(bucket=s3_bucket, key=s3_key)
-    s3_bootstrap_key = 'emr_bootstrap_train.sh'
+    # s3_bootstrap_key = 'emr_bootstrap_train.sh'
     s3_bootstrap_uri = 's3://{bucket}/emr_bootstrap_train.sh'.format(bucket=s3_bucket)
 
     # S3 bucket/key, where the spark job logs will be maintained
-    s3_log_bucket = config.DEPLOYMENT_PREFIX + '-automated-analytics-spark-jobs'
-    s3_log_key = '{}_{}_spark_emr_log_'.format(config.DEPLOYMENT_PREFIX, COMPONENT_PREFIX,
-                                               str_cur_time)
+    s3_log_bucket = 'cvae-insights'
+    s3_log_key = '{}_spark_emr_log_'.format(str_cur_time)
     s3_log_uri = 's3://{bucket}/{key}'.format(bucket=s3_log_bucket, key=s3_log_key)
 
     _logger.debug("Uploading the bootstrap action to AWS S3 URI {} ...".format(s3_bootstrap_uri))
 
     # Note: This overwrites if file already exists
-    s3_client = boto3.client('s3',
-                             aws_access_key_id=config.AWS_S3_ACCESS_KEY_ID,
-                             aws_secret_access_key=config.AWS_S3_SECRET_KEY_ID)
-    s3_client.upload_file(input_bootstrap_file, s3_bucket, s3_bootstrap_key)
+    # Manually stored bootstrap file in s3 bucket, so no need of s3 client.
+    # s3_client = boto3.client('s3',
+    #                          aws_access_key_id='',
+    #                          aws_secret_access_key='')
+    # s3_client.upload_file(input_bootstrap_file, s3_bucket, s3_bootstrap_key)
 
     _logger.debug("Uploading the src code to AWS S3 URI {} ...".format(s3_uri))
-    s3_client.upload_file(input_src_code_file, s3_bucket, s3_key)
+    # s3_client.upload_file(input_src_code_file, s3_bucket, s3_key)
 
     _logger.debug("Starting spark emr cluster and submitting the jobs ...")
     emr_client = boto3.client('emr',
-                              aws_access_key_id=config.AWS_S3_ACCESS_KEY_ID,
-                              aws_secret_access_key=config.AWS_S3_SECRET_KEY_ID,
+                              aws_access_key_id='',
```

Comment:
How does this work without a key? If you're not using it remove it entirely.

Reply:
No whoever will run the EMR cluster via this script will put the credentials. But yeah, I can pick it from the environment.